lambda/aws-rag-appsync-stepfn-opensearch/s3_file_transformer/src/requirements.txt

```diff
@@ -2,7 +2,9 @@ aws-lambda-powertools
 aws-xray-sdk
```

Comment:
missing deps versions

Reply:
will create a separate PR to add a version for this in all the constructs.